DigitalLearningSolutions.Web/Helpers/CommonValidationErrorMessages.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    public static class CommonValidationErrorMessages

```

Comment:
Could do with a better name, potensh.

This class is intended to house only validation error messages which
- Are sent after some logic is done in the app (so can't be set on data validation attributes on view models)
- Are not specific to the context (e.g. "Preview your new profile picture before saving" in MyAccountController)

Any bright ideas?